public-ui/src/app/data-browser/views/error-message/error-message-react.component.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { ClrIcon } from 'app/utils/clr-icon';
+import * as React from 'react';
+
+interface Props {
+    dataType: string;
+}
+
+export class ErrorMessageReactComponent extends React.Component<Props, {}> {
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  render() {
+    const {dataType} = this.props;
+    const iconClass = 'alert-icon';
+    const iconShape = 'exclamation-triangle';
+    const alertText =  (dataType === 'data') ? '<span className="alert-text"></span>' : '<span className="alert-text">' +
+                                'Sorry, the chart cannot be displayed. Please try refreshing the page.' +
+                              '</span>';
```

Comment:
This is currently unused. We should either call this in the template or remove it.

Reply:
Sorry this should not be in here.